File: Data Structures & Algorithms/two-integer-sum/submission-0.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {

        int[] results;

        for (int i=0; i<nums.Length;i++){
            for(int j=0; j<nums.Length;j++){
                if nums[i] + nums[j] == target{
                    results.Add(nums[i]);
                    results.Add(nums[j]);
                }
            }
        }

        return results;

    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        for(int i = 1; i < numbers.length; i++){
            if(numbers[i-1] + numbers[i] == target){
                return new int[] {numbers[i-1], numbers[i]};
            }
        }

        return new int[]{};
        
    }
}
